advent_of_code/2024/day06_part1.py

```
@@ -0,0 +1,101 @@
+        for index, line in enumerate(input_data):
+            current_line = line
+            if self.direction in line:
+                self.x = index
+                self.y = line.index(self.direction)
+                current_line = current_line.replace(self.direction, ".")
+
+            self.faculty_map.append(list(current_line))
+            self.x_max = len(self.faculty_map[0])
+            self.y_max = len(self.faculty_map)
+
+    def move(self) -> bool:
```

Ruff (0.12.2)

`move` is too complex (13 > 10)

(C901)